### DISCUSSION TITLE
2.0.6

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#10672] [General]
*  Fix issue with converting distributions from WSL1 to WSL2
*  Set ApplicationName when creating wslhost / wslrelay processes (solves #10651)
*  Remove spaces in g_ConfigBootInitWaitCommand (solves #10653)
*  Update MSRDC to 1.2.4677
*  Update Microsoft.Direct3D.Linux to version 1.611.1-81528511 (solves #10671)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/2.0.6'>2.0.6</a>.</em>

**Comment by MikolajGolowski** [comment]
Not all causes for #10651 were fixed in this update. Please see #10764
